cmd/thanos/bucket.go

```diff
@@ -375,6 +379,38 @@ func registerBucketWeb(m map[string]setupFunc, root *kingpin.CmdClause, name str
 	}
 }
 
+func registerBucketReplicate(m map[string]setupFunc, root *kingpin.CmdClause, name string, objStoreConfig *extflag.PathOrContent) {
+	cmd := root.Command("replicate", "Replicate data from one object storage to another. NOTE: Currently it works only with Thanos blocks (meta.json has to have Thanos metadata).")
+	httpMetricsBindAddr, _ := regHTTPFlags(cmd)
+	toObjStoreConfig := regCommonObjStoreFlags(cmd, "-to", false, "The object storage which replicate data to.")
+	resolution := cmd.Flag("resolution", "Only blocks with this resolution will be replicated.").Default(strconv.FormatInt(downsample.ResLevel0, 10)).Int64()
```

Comment:
I was thinking maybe it would be useful to do an `Enum` param here and list all of the three possible different options? Then it would be clearer to our users. Otherwise, they'd have to go into the code to know the possible values. Not sure if we have those millisecond values documented anywhere.

Reply:
It is int value indeed ...

Reply:
Comment in docs/components/compact.md i think, maybe not very clear.
I will use `Enum` here.